src/server/auth/server/roles.go

```diff
@@ -50,6 +50,7 @@ func init() {
 			auth.Permission_REPO_ADD_PIPELINE_READER,
 			auth.Permission_REPO_REMOVE_PIPELINE_READER,
 			auth.Permission_PIPELINE_LIST_JOB,
+			auth.Permission_CLUSTER_ENTERPRISE_PAUSE_STATUS,
```

Comment:
if we go with the `authenticated` approach mentioned, we can remove the `auth.Permission_CLUSTER_ENTERPRISE_PAUSE_STATUS`